<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelListWidget</class>
 <widget class="QWidget" name="ChannelListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="ChannelWidget" name="widget" native="true"/>
   </item>
   <item>
    <widget class="ChannelWidget" name="widget_2" native="true"/>
   </item>
   <item>
    <widget class="ChannelWidget" name="widget_8" native="true">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ChannelWidget" name="widget_7" native="true">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ChannelWidget" name="widget_3" native="true"/>
   </item>
   <item>
    <widget class="ChannelWidget" name="widget_5" native="true"/>
   </item>
   <item>
    <widget class="ChannelWidget" name="widget_4" native="true"/>
   </item>
   <item>
    <widget class="ChannelWidget" name="widget_6" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChannelWidget</class>
   <extends>QWidget</extends>
   <header>ChannelWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
